test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -192,9 +187,7 @@ protected ClassLoader getClassLoader() {
 
                                 Path location = testWiringClassesDir.resolve(resourceName);
                                 Files.createDirectories(location.getParent());
-                                try (FileOutputStream out = new FileOutputStream(location.toFile())) {
-                                    out.write(cw.toByteArray());
-                                }
+                                Files.write(location, cw.toByteArray());
```

Comment:
Are the changes to this file related?

Reply:
No, not at all, it is just a minor cleanup